.bazelrc

```diff
@@ -141,11 +142,11 @@ build:asan --copt -O1
 build:asan --copt -fno-optimize-sibling-calls
 
 # Clang ASAN/UBSAN
-build:clang-asan --config=clang
-build:clang-asan --config=asan
-build:clang-asan --linkopt -fuse-ld=lld
-build:clang-asan --linkopt --rtlib=compiler-rt
-build:clang-asan --linkopt --unwindlib=libgcc
+build:clang-asan --config=clang                                                             # unique
```

Comment:
As discussed offline, please wait for and pull https://github.com/envoyproxy/envoy/pull/33605 in. Let's keep this file as close to the Envoy's version as possible.

Reply:
Thanks.